PwshSpectreConsole/PwshSpectreConsole.psd1

```diff
@@ -58,7 +58,8 @@ RequiredAssemblies =
                '.\packages\Spectre.Console\lib\netstandard2.0\Spectre.Console.dll', 
                '.\packages\Spectre.Console.ImageSharp\lib\netstandard2.0\Spectre.Console.ImageSharp.dll', 
                '.\packages\SixLabors.ImageSharp\lib\netstandard2.0\SixLabors.ImageSharp.dll', 
-               '.\packages\Spectre.Console.Json\lib\netstandard2.0\Spectre.Console.Json.dll'
+               '.\packages\Spectre.Console.Json\lib\netstandard2.0\Spectre.Console.Json.dll',
+               '.\packages\Spectre.Console.Testing\lib\netstandard2.0\Spectre.Console.Testing.dll'
```

Comment:
Is that needed? is the testlib used in the public/private functions? 
i thought it was limited to pester tests which dont ship with the module

Reply:
I added it here to get it working for now. I think I'll add it with Add-Type in the @init test to import it unless there's a better way